spec/io/io_spec.rb

```diff
@@ -34,6 +34,16 @@
       end
     end
 
+    context ".from_csv_url" do
+      it "loads from a CSV file" do
+        df = Daru::DataFrame.from_csv_url('https://raw.githubusercontent.com/anshuman23/ruBayes/master/test/sample.csv')
```

Comment:
Typically, all requirements and configuration of testing libraries are done in `spec/spec_helper.rb`, and Daru is not an exception :)